### PR TITLE
delete "section below"

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/en-us/web/api/htmliframeelement/browsingtopics/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ browser-compat: api.HTMLIFrameElement.browsingTopics
 
 {{APIRef("HTML DOM")}}{{SeeCompatTable}}{{non-standard_header}}
 
-> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](/en-US/docs/Web/API/Topics_API#standards_positions) section below for details of opposition.
+> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](/en-US/docs/Web/API/Topics_API#standards_positions) for details of opposition.
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/browsingtopics/index.md:14]:
```suggestion
> **Warning:** This feature is currently opposed by two browser vendors. See the [Standards positions](/en-US/docs/Web/API/Topics_API#standards_positions) section for details of opposition.
```

Comment (T34-active) [reply files/en-us/web/api/htmliframeelement/browsingtopics/index.md:14]: Thank you! i need this You've been a great help.👍

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by T34-active** [comment]
appreciate

**Pull request merged**